test-requirements.txt

```diff
@@ -0,0 +1,5 @@
+# List required packages in this file, one per line.
+bluesky
+event-model
+json
+tifffile
```

Comment:
I think this is missing `ophyd`. Also, this should not include anything that is already included in `requirements.txt`. It does no harm to double up from a technical point of view, but it is confusing to human readers try to evaluate what the dependency load is.